src/CoreWCF.ConfigurationManager/src/CoreWCF/Configuration/WSHttpBindingCollectionElement.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace CoreWCF.Configuration
 {
-    public class WSHttpBindingCollectionElement : StandardBindingCollectionElement<WSHttpBinding, WSHttpBindingElement>
+    public class WsHttpBindingCollectionElement : StandardBindingCollectionElement<WSHttpBinding, WsHttpBindingElement>
     {
```

Comment:
The WS is usually both uppercase as they are initials for Web Services so are each technically like a separate word. So WSHttpBinding has both W and S uppercase. Can you change this back to match .NET Framework and the other classes using WS and not Ws.

Reply:
Unfortunately this was inadvertently caused by the utility I used to fix the naming of the private variables, and I didn't catch it.